shared/src/main/scala/Elements.scala

```diff
@@ -287,6 +288,21 @@ object Elements:
         case list: VList => list.vmap(execHelper)
         case _ => throw new Exception("Can't exec on functions in lists")
 
+    val execNotPop = addDirect(
+      "Ḃ",
+      "Execute lambda without popping | Evaluate as Vyxal without popping",
```

Comment:
```suggestion
      "Execute lambda without popping arguments | Evaluate as Vyxal without popping arguments",
```

Reply:
This is only a temporary element for testing anyway. The description was copilot generated and if anything, this will probably be a digraph

Reply:
Ah, ok